src/main/app/forms/models/statementOfTruth.ts

```diff
@@ -0,0 +1,46 @@
+import { IsDefined, MaxLength } from 'class-validator'
+
+import { IsNotBlank } from 'forms/validation/validators/isBlank'
+
+import { Serializable } from 'models/serializable'
+
+export class ValidationErrors {
+  static readonly SIGNER_NAME_REQUIRED: string = 'Enter the name of the person signing the statement'
+  static readonly CONTENT_TOO_LONG: string = 'You’ve entered too many characters'
```

Comment:
just minor suggestion, I would prefer to have this named like `SIGNEE_ROLE_TOO_LONG`